workflows/templates/sync-template.yaml

```diff
@@ -36,6 +36,8 @@ spec:
             value: "{{inputs.parameters.THOTH_FORCE_SYNC}}"
           - name: "THOTH_LOG_GRAPH_SYNC_JOB"
             value: "DEBUG"
+          - name: "THOTH_SYNC_INSPECTION_ONLY_GRAPH"
```

Comment:
What does it do? The name is not really speaking to me ;)

Reply:
It will aggregate results and upload them to ceph. But syncing logic from ceph to graph will not happen. This should be replaced over time with argo handling JSON files produced by inspections.

BTW why is it there if this is solver specific sync?

Reply:
In this case the sync on Ceph is done by the previous task, so graph-sync-job just need to sync to graph database. Don't we have to provide `THOTH_ONLY_{component}_DOCUMENTS` variable which is different for each service?

Reply:
Parameters with `THOTH_ONLY_*` are used only if you do not provide document to be synced explicitly - then all documents are considered and `THOTH_ONLY_*` picks just the desired document types.